distage/distage-framework-docker/src/main/scala/izumi/distage/docker/Docker.scala

```diff
@@ -194,13 +198,16 @@ object Docker {
     *
     * @param registry     Options to connect to custom Docker Registry host,
     *                     will try to connect to specified registry, instead of the default if [[useRegistry]] is `true`
+    *
+    * @param searchTag    A tag that is used to search containter. Useful when several containers are available to reuse
     */
   final case class ClientConfig(
     globalReuse: DockerReusePolicy = ClientConfig.defaultReusePolicy,
     useRemote: Boolean = false,
     useRegistry: Boolean = false,
     remote: Option[RemoteDockerConfig] = None,
     registry: Option[DockerRegistryConfig] = None,
+    searchTag: Option[String] = None,
   )
```

Comment:
Don't like this, but I actually haven't find a better way to pass a client search tag yet :/